static/app/views/dashboards/widgets/timeSeriesWidget/timeSeriesWidgetVisualization.tsx

```diff
@@ -78,6 +82,7 @@ export interface TimeSeriesWidgetVisualizationProps {
    * Array of `Release` objects. If provided, they are plotted on line and area visualizations as vertical lines
    */
   releases?: Release[];
+  showReleaseLines?: boolean;
```

Comment:
What do you think about a string union here? `showReleaseLines` seems like it turns something on/off rather than toggling between lines and bubbles. Other Dashboards widgets usually use string unions. Here, it would be something like `showReleaseAs?: 'bubbles' | 'lines'`

Reply:
[`9697da7` (#85270)](https://github.com/getsentry/sentry/pull/85270/commits/9697da7c283b1b0cf27ae355ddf12ef4d904fdfd)